content/en/security/cloud_siem/guide/google-cloud-config-guide-for-cloud-siem.md

```diff
@@ -0,0 +1,109 @@
+1. Click **Save**. 
+
+### Change default configuration for new services
+
+If a new Google Cloud service is added, it inherits your [default audit configuration][7].
+
+To ensure that Data Access audit logs are captured for new Google Cloud services, modify your default audit configuration:
+
+1. Navigate to the **IAM & Admin Console > [Audit Log][6]**.
+1. Enable **Admin Read**, **Data Read**, and **Data Write**.
+1. Click **Save**.
+
+## Create a new Google Cloud publish/subscription (pub/sub) system
+
+1. Navigate to the Pub/Sub > [Topics][7].
+1. Click **Create Topic**.
+1. Enter a topic name. For example, `export-audit-logs-to-datadog`.
+1. Click **Create**.
+
+## Configure the pub/sub to send logs to Datadog
+
+1. Navigate to Pub/Sub > [Subscriptions][8].
+1. Click **Create Subscription**.
+1. Enter a subscription name.
+1. Select the topic you created earlier.
+1. In **Delivery type**, select **Push**.
+1. Enter the following endpoint URL, replacing `DATADOG_API_KEY` with an existing or new Datadog API key: `https://gcp-intake.logs.{{< region-param key="dd_site" code="true" >}}/api/v2/logs?dd-api-key=<DATADOG_API_KEY>&dd-protocol=gcp`
+1. Configure any additional options for your use case.
+1. Click **Create**.
+
+The pub/sub is ready to receive logs from Google Cloud Logging and forward them to Datadog.
+
+## Export logs from Google Cloud Logging to the pub/sub
+
+1. Navigate to [Cloud Logs Explorer][9].
+1. Enter the query to filter to the logs you want to export.
+1. Select **Log Router** in the left side menu.
+1. Click **Create Sink**.
+1. Enter a name for the sink.
+1. Click **Next**.
+1. In the **Select Sink Service** dropdown menu, select **Cloud Pub/Sub topic**. 
+1. In the **Select a Cloud Pub/Sub topic**, select the Pub/Sub created earlier.
+1. Click **Create Sink**.
+
+**Note**: It is possible to create several exports from Google Cloud Logging to the same pub/sub with different sinks.
+
+**Warning**: Pub/subs are subject to [Google Cloud quotas and limitations][10]. If the number of logs you have is higher than those limitations, Datadog recommends you split your logs over several topics. See [Monitor the Log Forwarding][11] for information on how to set up a monitor to notify when you are close to those limits.
+## Use Cloud SIEM to triage Security Signals
+
+Cloud SIEM applies out-of-the-box detection rules to all processed logs, including the Google Cloud audit logs you have just set up. When a threat is detected with a detection rule, a Security Signal is generated and can be viewed in the Security Signals Explorer.
+
+- Go to the [Cloud SIEM Signals Explorer][12] to view and triage threats. See Security Signals Explorer for further details.
+- You can also use the [Google Cloud Audit Log dashboard][13] to investigate anomalous activity.
+- See [out-of-the-box detection rules][14] that are applied to your logs.
+- Create [new rules][15] to detect threats that match your specific use case.
+
+## Further reading
```

Comment:
should we link to the Google cloud docs here as well? https://docs.datadoghq.com/integrations/google_cloud_platform/#log-collection

Reply:
Good idea, added it